
string name
string description
Machine pc
KeyValue[] values